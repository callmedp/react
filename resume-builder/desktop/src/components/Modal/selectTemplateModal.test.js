import { shallow } from 'enzyme';
import React from 'react';
import SelectTemplateModal from './selectTemplateModal';


describe('SelectTemplateModal', () => {

    it('Select Template Modal debug', () => {
        const  props ={
            ui: {
                'select_template_modal': true 
            }
        },
         ModalWrapperComponent = shallow(<SelectTemplateModal {...props} />);
        expect(ModalWrapperComponent).toMatchSnapshot();
    });
})

