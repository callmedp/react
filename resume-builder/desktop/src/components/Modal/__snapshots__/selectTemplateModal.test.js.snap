// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectTemplateModal Select Template Modal Test 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="Modal"
  closeTimeoutMS={0}
  contentLabel="onRequestClose"
  isOpen={true}
  overlayClassName="Overlay"
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  role="dialog"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <ResumeSlider
    ui={
      Object {
        "select_template_modal": true,
      }
    }
  />
  <button
    className="Modal--close"
    onClick={[Function]}
  >
    +
  </button>
</Modal>
`;

exports[`SelectTemplateModal Select Template Modal debug 1`] = `
<Modal
  ariaHideApp={true}
  bodyOpenClassName="ReactModal__Body--open"
  className="Modal"
  closeTimeoutMS={0}
  contentLabel="onRequestClose"
  isOpen={true}
  overlayClassName="Overlay"
  parentSelector={[Function]}
  portalClassName="ReactModalPortal"
  role="dialog"
  shouldCloseOnEsc={true}
  shouldCloseOnOverlayClick={true}
  shouldFocusAfterRender={true}
  shouldReturnFocusAfterClose={true}
>
  <ResumeSlider
    ui={
      Object {
        "select_template_modal": true,
      }
    }
  />
  <button
    className="Modal--close"
    onClick={[Function]}
  >
    +
  </button>
</Modal>
`;
